Twig/Extension/SonataAdminExtension.php

```diff
@@ -171,6 +174,8 @@ public function renderListElement(
     }
 
     /**
+     * @internal
```

Comment:
> Maybe just test real public method that use those ones? :wink:

That's what I would do if I was starting from zero, but I am not it would be too much work I think.